meetpy/meetups/views.py

```diff
@@ -15,9 +17,45 @@ class MeetupsListView(generic.ListView):
     model = models.Meetup
 
 
-class MeetupDetailView(generic.DetailView):
-    model = models.Meetup
-    slug_url_kwarg = slug_field = 'number'
+class MeetupRedirectOrList(generic.View):
```

Comment:
Why not TemplateView?

Reply:
It's simpler. Using features provided by TemplateView would require overriding `get_context_data` as well, then either assign `meetups` somewhere to `self`, or query again, then call `super().get()`... using `TemplateResponse` directly is simpler and if we're not using things from TemplateView then there's no point in inheriting from it. :)